query.py

```diff
@@ -437,9 +439,23 @@ def init_class(cls) -> None:
         cls._tree_processors = [t[1] for t in sorted(tree_procs, key=lambda x: -x[0])]
         cls._text_processors = [t[1] for t in sorted(text_procs, key=lambda x: -x[0])]
 
-        # Obtain query grammar fragments from the tree processors
+        # Query grammar plug-ins:
+        # Additional grammar fragments,
+        # which can be used in query module grammars
+        cls._query_grammar_additions = []
+        plugnames = modules_in_dir("queries/plugins")
```

Comment:
(We may want to define the path "queries/plugins" as a constant somewhere, I just hardcoded it for the time being as this is only an idea.)

Reply:
Yes. a constant at the beginning of the file would be better. Also, double-check that the forward slash is OK here, as this may potentially run on Windows which uses backslashes. A safe way is to use `os.path.join("queries", "plugins")`.